src/views/TournamentDetail.vue

```diff
@@ -1,573 +1,96 @@
+  swiss: { title: 'Système Suisse', is_available: tournament.value?.swissRounds.length > 0 || false },
+  brackets: { title: 'Arbres', is_available: tournament.value?.brackets.length > 0 || false },
+  planning: { title: 'Planning', is_available: true },
+  rules: { title: 'Règlement', is_available: true },
+}));
 
-const swiss_match_results = (matchs : SwissMatch[]) => {
-  const round_matchs = groupBy(matchs, 'round_number');
-  const res = {} as Record<string, Record<string, string | Record<string, string | number | boolean | undefined>[]>[]>;
-  Object.entries(round_matchs).forEach(([round, roundMatchs]) => {
-    res[round] = [] as Record<string, string | Record<string, string | number | boolean | undefined>[]>[];
-    roundMatchs.forEach((match) => {
-      const tmp = {} as { teams: Record<string, string | number | boolean | undefined>[]; status: string };
-      tmp.teams = [];
-      tmp.status = match.status;
-      match.teams.forEach((team) => {
-        const data = {} as { name: string | undefined; score: number; is_winner: boolean };
-        data.name = get_validated_team_by_id(team)?.name;
-        data.score = match.score[team];
-        data.is_winner = is_winning_team(match, team);
-        tmp.teams.push(data);
-      });
-      res[round].push(tmp);
-    });
-  });
+const router = useRouter();
+const route = useRoute();
 
-  return res;
-};
+const selected_section = ref<string>(route.path.split('/').at(-1));
 
-const router = useRouter();
 try {
   await getTournamentFull(props.id);
   getTournamentTeams();
 } catch (err: unknown) {
   router.go(-1);
 }
-
-if (props.section !== undefined && props.section.s in sections) {
-  sections[props.section.s][0] = true;
-  drop_label.value = props.section.s;
-  trans.value = `translateX(calc(-100vw * ${sections[props.section.s][1]}`;
-} else {
-  sections.info[0] = true;
-}
 </script>
 
 <template>
-  <div v-if="tournament?.is_announced" class="flex min-h-[calc(100vh_-_6rem)] flex-col items-center">
+  <div v-if="tournament?.is_announced" class="min-h-[calc(100vh_-_6rem)]">
     <div class="sticky top-24 min-w-full bg-[#2c292d]">
       <div class="text-center text-6xl font-bold text-white">
         {{ tournament?.name }}
       </div>
 
-      <nav class="mt-2 flex justify-center bg-gray-500 py-4">
+      <nav class="mt-2 flex justify-center gap-16 bg-gray-500 py-4">
         <button
-          :class="{ 'after:translate-x-[5px] after:rotate-90 ': open_drop, 'after:-rotate-90': !open_drop }"
-          class="text-xl after:inline-block after:w-4 after:origin-center after:content-['\2039'] md:hidden"
           type="button"
-          @click="open_drop = !open_drop"
+          class="text-xl underline decoration-[#63d1ff] decoration-4 underline-offset-8 md:hidden"
+          @click="open_dropdown = !open_dropdown"
         >
-          {{ drop_label }}
-        </button> <!-- id="dropdown-btn"--> <!--@click="switch_tag"-->
+          {{ sections[selected_section].title }}
```

Comment:
If the url does not include a subsection (for example, `/tournament/6`), there is an error. You should fallback on the first one in that case.

Reply:
Yes, I thought about it after but didn't get the time to push the change '^^